Public/Src/Engine/Scheduler/Tracing/Log.cs

```diff
@@ -1294,6 +1294,20 @@ internal static string PipWriteFileFailedMessage(string pipDescription, string p
                 EventConstants.PipPrefix + "Process is going to be retried due to exiting with exit code '{exitCode}' (remaining retries is {remainingRetries})")]
         public abstract void PipWillBeRetriedDueToExitCode(LoggingContext context, long pipSemiStableHash, string pipDescription, int exitCode, int remainingRetries);
 
+        [GeneratedEvent(
+            (int)EventId.PipRetry,
+            EventGenerators = EventGenerators.LocalOnly,
+            EventLevel = Level.Verbose,
+            Keywords = (int)(Keywords.UserMessage | Keywords.UserError),
+            EventTask = (int)Tasks.PipExecutor,
+            Message = EventConstants.PipSpecPrefix + " - Retry failed process")]
+        public abstract void PipProcessRetry(

```

Comment:
>PipProcessRetry [](start = 29, length = 15)

It would be good to include the stdout/stderr along with the exit code to show why it is being retried